### PR TITLE
Simplify download options & layout

### PR DESCRIPTION
**Pull request by alex-deaconu**
Show Settings on Save
Change settings position

Tracking for Player Downloads added
Show How-To on Display Setup

[stage-3]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks.

File: web/partials/displays/display-fields.html [hunk 1]
```diff
@@ -14,116 +14,46 @@
...
-      </div>
-    </div>
-    <div class="col-md-3">
-      <div class="form-group">
-        <label class="control-label" translate>displays-app.fields.address.postalCode</label>
-        <input type="text" class="form-control" ng-model="display.postalCode">
-      </div>
+  <div class="panel panel-success display-panel"> 
+    <div class="panel-heading"> 
+      <h3 class="panel-title"> Your Display ID is </h3>
```
Comment (ezequielc) [inline web/partials/displays/display-fields.html]: @alex-deaconu Should we externalize these strings? There is another one below, 'Install Rise Player'. Thanks

Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @ezequielc good call. Will fix. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu There are 2 or 3 strings without i18n in `display-fields.html`. Otherwise looks good. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc fixed the i18n strings, in the second commit. Will squash once you review. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Looks good :+1:  Thanks for fixing it.

**Pull request merged**